server/public/model/support_packet.go

```diff
@@ -27,6 +27,8 @@ type SupportPacketDiagnostics struct {
 	Server struct {
 		OS               string `yaml:"os"`
 		Architecture     string `yaml:"architecture"`
+		CPUCores         int    `yaml:"cpu_cores"`
+		TotalMemory      uint64 `yaml:"total_memory"`
```

Comment:
Should use explicit name like TotalMemoryMB, otherwise only "bytes" seems reasonable without any unit.